src/layout/Tabbed.js

```diff
@@ -61,14 +62,32 @@
 
     Tabbed.prototype.enter = function (domNode, element) {
         HTMLWidget.prototype.enter.apply(this, arguments);
-        this._tabContainer = element.append("div");
         this._contentContainer = element.append("div");
+        this._contentContainer.attr("id", "content_container");
     };
 
     Tabbed.prototype.update = function (domNode, element) {
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
 
+        // if (this._tabContainer) {this._tabContainer.remove();}
+        // this._tabContainer = element.insert("div", this.tabLocation() === "bottom" ? null : "div");
+        // this._tabContainer.attr("class", "on_" + this.tabLocation());
+
+        this._tabContainer = element.selectAll("#tab_contanier").data([this.tabLocation()], function(d) {return d;});
+        this._tabContainer
+            .enter()
+            .insert("div", function(d1) {
+                var x = d1 === "top" ? "#content_container" : null;
+                console.log(x, d1);
+                return x;
+            })
```

Comment:
@GordonSmith I tried this approach which seemed logical, however it is not working as I expected.  When the tabLocation is set to "bottom" it works when the insert function returns "null".  But when it is set to "top" (use the property editor) and returns "#content_container" it fails.  If I remove the function and hard code "#content_container" it works as expected.  Any thoughts?

http://rawgit.com/dtsnell4/Visualization/GH-1148_Bottom_Tabs/demos/dermatology.html?src/layout/Tabbed